deepchem/utils/test/test_misc_utils.py

```diff
@@ -0,0 +1,8 @@
+def test_uniquifier():
+    from deepchem.utils import Uniquifier
+    a = 1
+    b = 2
+    c = 3
+    d = 1
+    u = Uniquifier([a, b, c, a, d])
+    u.get_unique_objs() == [1, 2, 3]
```

Comment:
@GreatRSingh This isn't an assert. You need to add an assert there or this will just always passs. Please add the assert in your next PR